src/main/java/com/icthh/xm/ms/balance/service/BalanceSpecService.java

```diff
@@ -93,4 +98,18 @@ public void onInit(String configKey, String configValue) {
     public BalanceSpec.BalanceTypeSpec getBalanceSpec(String typeKey) {
         return getTypeSpecs().getOrDefault(typeKey, NULL_OBJECT);
     }
+    public List<String> balanceStatusKeys(String balanceTypeKey) {
```

Comment:
there should be an empty line between methods

Reply:
fixed